### PR TITLE
Uniformize tests with the new API

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-core#716

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 3.6.O - [#66](https://github.com/openfisca/country-template/pull/66)
```
Comment (maukoquiroga) [inline CHANGELOG.md]: Isn't it a breaking change? (I'll approve if we agree it is not, or if we bump major)

Comment (fpagnoux) [reply CHANGELOG.md]: Which usecase are you thinking about?

Comment (fpagnoux) [reply CHANGELOG.md]: I guess we could consider that a BC if we consider that users can run tests of their own that are not part of the package 🤔

Would you have time to suggest or add a few commits to explicit the breaking change?

Comment (maukoquiroga) [reply CHANGELOG.md]: Hey, after thinking a bit in the metro, and by the way the library is used, I guess you're right is a minor.

**Pull request merged**